_sass/components/_footnotes.scss

```diff
@@ -31,19 +31,19 @@ body {
   ol {
     @include heading('para-sm');
 
-    font-weight: 200;
+    font-weight: 400;
     list-style-type: decimal;
-    padding-bottom: $base-padding;
     padding-left: $base-padding;
-    padding-top: $base-padding-lite;
 
     &:target {
       background-color: $mid-blue;
     }
   }
 
   p {
-    padding-bottom: $base-padding-lite;
+    @include heading('para-med');
```

Comment:
I think this is [`@include heading('para-med');`](https://github.com/18F/doi-extractives-data/blob/bb6b321ed9d83493867696e905ae2461c0c2d115/_sass/mixins/_type-mixins.scss#L75)